index.html

```diff
@@ -1226,7 +1226,7 @@ <h2>DID Subject</h2>
       <dl>
         <dt><dfn>id</dfn></dt>
         <dd>
-The value of <code>id</code> MUST be a single valid <a>DID</a>.
+The value of <code>id</code> MUST be a single valid <a>DID</a>. 
```

Comment:
Remove trailing whitespace.
```suggestion
The value of <code>id</code> MUST be a single valid <a>DID</a>.
```